sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/implementation/GroupableParentResourceWithTagsImpl.java

```diff
@@ -23,7 +24,7 @@
         InnerModelT extends com.azure.core.management.Resource,
         FluentModelImplT extends
             GroupableParentResourceWithTagsImpl<FluentModelT, InnerModelT, FluentModelImplT, ManagerT>,
-        ManagerT extends ManagerBase>
+        ManagerT extends Manager<? extends AzureServiceClient>>
```

Comment:
Not a big deal. Is `AzureServiceClient` needed? If no need prefer just a `<?>`.

Reply:
done